content/blog/2020/2020-01-29-machine-anthropology.md

```diff
@@ -0,0 +1,44 @@
+
+Related: {{< link_new url="hblog/2018/11/06/data-mining-for-anthropologists/" name="Data Mining for Anthropologists">}}
+
+{{< figure src="/blog_img/2020/2020-01-29-MA-big.png" >}}
+
+Copenhagen Center for Social Data Science ([SODAS](https://sodas.ku.dk/)) held a conference on machine anthropology just a few days ago and it was inspiring to see how the humanities benefit from computer sciences and vice versa! In our talk I presented my PhD research of sensor data from a smart building and used Orange to show how to detect patterns of behaviour in such data.
+
+Let us use Philadelphia Crime data from the Datasets widget. We will have to create two new features, one for hour of the day and one for month. In the Feature Constructor we will write two Python expressions that parse month and hour from the datetime feature.
+
+{{< figure src="/blog_img/2020/2020-01-29-feat-const.png" >}}
+
+How does our data look like now?
+
+{{< figure src="/blog_img/2020/2020-01-29-data-table.png" >}}
+
+Perfect. Now say we would like to establish the frequency of crimes by the hour. We will use Pivot Table to count the occurrences of each type of crime. We set rows to type, because we are interested in the differences between different types of crimes. Then we set hour as columns. This will create a new timeseries-like data table, where each columns will represent a single hour of the day. Values can be set to anything, because we will not use any special type of aggregation, but simply count occurrences of crimes.
+
+{{< figure src="/blog_img/2020/2020-01-29-pivot-table.png" >}}
+
+We can observe the results of the pivot table in a Line Plot. We group by type and see how crime frequencies changes with respect to the hour of the day. Unsurprisingly, the liquor law violations and prostitution go up late at night. Prostitution in particular, is a very frequent crime in Philadelphia (or at least was in the time of reporting).
+
+{{< figure src="/blog_img/2020/2020-01-29-line-plot.png" >}}
+
+If we standardize the data with Preprocess (default normalization option), we see a more balanced picture, where homicides are relatively much more frequent early in the morning than at any other time of the day.
+
+{{< figure src="/blog_img/2020/2020-01-29-line-plot-standardize.png" >}}
+
+There you are, a workflow for observing simple timeseries patterns in the data. Of course, you can create much more complicated workflows in Orange or even write a custom Python script, but for now, this is it. If you have you own examples of anthropological, sociological, or any kind of socially-oriented data analysis in Orange, we'd love to hear about it!
```

Comment:
"but for now, this is it": I think this may be a bit redundant.

Reply:
Yeah, I love this phrase just too much. It's a goner.